go/ql/test/library-tests/semmle/go/dataflow/ExternalFlowInheritance/mad_s1_subtypes_true.expected

```diff
@@ -0,0 +1,10 @@
+invalidModelRow
+paths
+| test.go:20:7:20:16 | call to Source | test.go:22:9:22:9 | y |
```

Comment:
Currently subtypes = true for a non-interface does nothing; if it did anything in the future it would be to include embeddings, which we do here by accident